base/pointer.jl

```diff
@@ -60,11 +60,13 @@ end
 unsafe_wrap{N,I<:Integer}(Atype::Type, p::Ptr, dims::NTuple{N,I}, own::Bool=false) =
     unsafe_wrap(Atype, p, convert(Tuple{Vararg{Int}}, dims), own)
 
-unsafe_load(p::Ptr,i::Integer) = pointerref(p, Int(i))
+unsafe_load(p::Ptr, i::Integer, align::Integer) = pointerref(p, Int(i), Int(align))
```

Comment:
Just mimicking the existing definitions (for `i::Integer`). But yeah, that does make sense.